File: dataset/avatar/Python/Code/atcoder_ABC178_A.py
import os
from sys import stdin , stdout
def solve ( tc ) :
    a = int ( stdin.readline ( ).strip ( ) )
    print ( 1 if a == 0 else 0 )
tcs = 1
tc = 1
while tc <= tcs :
    solve ( tc )
    tc += 1

File: output/gpt_4o/avatar/direct_translation/python/java/successful_translation/atcoder_ABC178_A.java

import java.io.BufferedReader;
import java.io.InputStreamReader;
import java.io.IOException;

public class atcoder_ABC178_A {
    public static void main(String[] args) throws IOException {
        BufferedReader stdin = new BufferedReader(new InputStreamReader(System.in));
        int tcs = 1;
        int tc = 1;
        while (tc <= tcs) {
            solve(tc, stdin);
            tc++;
        }
    }

    public static void solve(int tc, BufferedReader stdin) throws IOException {
        int a = Integer.parseInt(stdin.readLine().trim());
        System.out.println(a == 0 ? 1 : 0);
    }
}

